scss/_utilities.scss

```diff
@@ -602,14 +615,15 @@ $utilities: map-merge(
       values: map-merge(
         $utilities-bg-colors,
         (
-          "body": $body-bg,
-          "white": $white,
```

Comment:
Just noticing that this class is still there and use `--bs-white-rgb` instead of `--bs-white-bg-rgb` as the others in this map ?

Reply:
Rethinking of this, since it represents a color, it might be fine as-is.